Cargo.toml

```diff
@@ -5,12 +5,19 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+description = "P2P network crawler data cruncher for graph metrics"
+readme = "Readme.md"
+
 [dependencies]
+anyhow = "1.0"
```

Comment:
yes! I would like to use it everywhere eventually

Reply:
We're using it like here:
https://github.com/runziggurat/crunchy/blob/d29427f6f202ae394e1c32cabfb187691e067585/src/config.rs#L48

That was previous PR where only config.rs file was commited without enabling crunchy to use it  (that's why Cargo was not updated to include deps not used by the app). I've splitted to get smaller PRs.

Reply:
Until it's not enabled in `main.rs` (as `mod config;`) compiler won't complain. I've not enabled it as it would result in used nowhere code - now it's enabled (as we're using it) so it need to be in Cargo.toml :)